src/app/shared/components/template/components/text/text.component.scss

```diff
@@ -0,0 +1,7 @@
+[data-variant~="caption"] {
+  font-size: var(--font-size-text-small);
+  text-align: center;
+  position: relative;
```

Comment:
This seemed more straightforward than trying to override the multiple levels of margin applied to the nest of elements.